core/src/components/input/test/input.spec.ts

```diff
@@ -44,3 +44,57 @@ describe('input: rendering', () => {
     expect(bottomContent).toBe(null);
   });
 });
+
+/**
+ * Input uses emulated slots, so the internal
+ * behavior will not exactly match IonSelect's slots.
```

Comment:
Nitpick: can we keep the way we refer to components consistent, i.e. `Input` and `Select` or `IonInput` and `IonSelect`

```suggestion
 * behavior will not exactly match Select's slots.
```